src/com/vinnik/kate/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.vinnik.kate;
+
+/** Class, which implements interaction with user. */
+public class Main {
+    public static void main(String[] args) {
+        try {
+            int amountOfThreads = Integer.parseInt(args[0]);
+            MultithreadingSorter.measureTimeOfSorting(amountOfThreads);
+        } catch (java.lang.ArrayIndexOutOfBoundsException e) {
```

Comment:
You're handling index out of bound situations, but what'd happen it a string value would be passed to your program as an argument? Or an integer value that's above Integer's maximum value? For example, I tried `129312892389981328919828913` as an input and got an unhandled exception.
Please, handle those situations with an appropriate message that'd describe the usage of the program to the user.

Reply:
Also, think about default value for amount of threads - how much would you set and why?

Reply:
It seems to me, that it is necessary to measure time of sorting "big array" - union of arrays (it's size = 500 * <amountThreads>, amountOfThreads is incrementing) and measure time of sorting by threads. When multithreading sort will become faster, than simple sort, we will figure out the best default number of threads.